client/src/styles/05-objects/_objects.wrappers.scss

```diff
@@ -0,0 +1,4 @@
+.o-page-content-wrapper {
+    max-width: 1200px;
```

Comment:
Why do we need this max-width ?

Reply:
Because the maximum width is specified in our design guideline in Figma. 
1. Open this site: https://stackoverflow.com/ and zoom out 4, 5 times. When you zoom out you can see that the content gets smaller and smaller but stays in the middle of the page. That's why we use "max-width" in this case. 
2. Now you can zoom out in Github and see what happens with the navbar...

Reply:
@ANeshkova Thanks for the clarification